src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs

```diff
@@ -362,6 +363,19 @@ public bool EnableMultipleHttp2Connections
         internal bool SupportsProxy => true;
         internal bool SupportsRedirectConfiguration => true;
 
+        /// <summary>
+        /// When non-null, a custom callback used to open new connections.
+        /// </summary>
+        public Func<SocketsHttpConnectionContext, CancellationToken, ValueTask<Stream>>? ConnectCallback
```

Comment:
Needs an XML comment.

Reply:
It has one.

Reply:
So it does :)

Reply:
I assume you were just so shocked that I remembered to add an XML comment, that you missed it :)